### DISCUSSION TITLE
How is the layout "Hands Down gold" preforming better in spanish text?

### DISCUSSION THREAD

**Discussion by masters3d** [binarybottle/engram-es#41] [Q&A]
https://www.reddit.com/r/ErgoMechKeyboards/comments/xrujby/comment/iqi61lm/?utm_source=share&utm_medium=web2x&context=3

https://github.com/masters3d/keyboard_layout_config_mapper/blob/8e027f767fe1c0b9c8b9cab2c4268e167eceed8e/layouts/Hands_Down_gold.ergo.json

![image](https://user-images.githubusercontent.com/6539412/193356573-fcddc9b4-26a8-4c18-a730-cd56bbe1909e.png)

https://klatest.keyboard-design.com/#/results

![image](https://user-images.githubusercontent.com/6539412/193356757-8e8d412b-1cfe-42f3-b572-34e7e503587c.png)
![image](https://user-images.githubusercontent.com/6539412/193356798-701dad48-dffc-45a8-92d1-b855ccf3989a.png)

**Comment by binarybottle** [comment]
Hello, @masters3d -- thank you for pointing us to the "Hands down" layout family!

Two notes:

- Ranking layouts is very sensitive to the measures and corpus you use to compare them.  Which measures are of greatest interest to you, and what type of (Spanish) text is representative of what you would type?

- I wouldn't compare layouts with letters on the thumb pad with layouts that do not -- different ballgame altogether.

**Reply by masters3d** [reply]
Makes sense. I did modify the gold layout to not use thumbs keys and still did somewhat better using the default Spanish option on the above testing site. 

The target Spanish i am thinking is probably closer to [Reina Valera Actualizada](http://simoncozens.github.io/open-source-bible-data/). I haven't done any testing yet on that. 

I was just surprise that a non Spanish layout would beat a Spanish one. After further reading I think the "hands down" layout uses some [Japanese](https://en.m.wikipedia.org/wiki/Romanization_of_Japanese) in its training data which might have many similarities to Spanish. 

engram-es also is surprising in other good ways. It's the best layout for English "The little prince".

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo, including Engram-es for Spanish!
